<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title> Alejandro Rodríguez-Gijón | Personal website </title>
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
  <link href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css" rel="stylesheet">
</head>
<body>

  <div class="container py-5">
  <h1>Bioinformatic tools</h1>
  <p>These are some scripts and tools that I have created and used! All of them can be found in my GitHub.</p>

  <!-- Tools list -->
<!-- Publication 1 -->
<div class="col">
  <div class="card bg-white text-dark border border-dark shadow-sm">
    <div class="card-body">
      <h5 class="card-title">Competitive mapping</h5>
      <p class="card-text mb-1">These scripts were used in: Rodríguez-Gijón A, et al. Widely-distributed freshwater microorganisms with streamlined genomes co-occur in cohorts with high abundance.</p>
      <p class="card-text text-muted">Scientific Reports 15, 34482 (2025). DOI: <a href=" https://doi.org/10.1038/s41598-025-22383-7" target="_blank" class="text-decoration-none">10.1038/s41598-025-22383-7</a>.</p>
      <p class="card-text mb-1">Link to the GitHub Repository: <a href="https://github.com/alejandrorgijon/competitive_mapping_scripts" target="_blank" class="text-decoration-none">https://github.com/alejandrorgijon/competitive_mapping_scripts</a>.</p>
      <p class="card-text mb-1"><strong>Summary</strong>: These scripts were used to estimate the relative abundance of a series of species-level clusters of Archaea and Bacteria across a set of raw metagenomic reads. These scripts help you 1) to download the FastQ files of the metagenomic reads, 2) to trim the metagenomic reads leveraging MiGA and various trimming tools, 3) to map the selected representative genomes against the trimmed metagenomic reads, 4) to estimate the number of prokaryotic genomes present on the set of trimmed reads, and 5) to estimate the relative abundance of each representative genome or species-level clusters.</p>
    </div>
  </div>
</div>
<p></p>
    
<!-- Publication 2 -->
<div class="col">
  <div class="card bg-white text-dark border border-dark shadow-sm">
    <div class="card-body">
      <h5 class="card-title">Plada (R package)</h5>
      <p class="card-text mb-1">Link to the GitHub Repository: <a href="https://github.com/alejandrorgijon/plada_package" target="_blank" class="text-decoration-none">https://github.com/alejandrorgijon/plada_package</a>.</p>
      <p class="card-text mb-1"><strong>Summary</strong>: This R package was originally designed to help the user to parse metagenomic data, but it can be used across a diversity of projects and data types. To install it and use it, please check the GitHub repository indicated above.</p>
    </div>
  </div>
</div>

  <!-- Contact section -->
  <div class="row mt-5">
    <div class="col-md-8">
      <h2>Contact</h2>
      <p>If you would like to contact me, you can write me to <a href="mailto:alejandro.rgijon@mncn.csic.es">alejandro.rgijon@mncn.csic.es</a></p>
      <a></a>
      <a href="index.html" class="btn btn-secondary mt-3">← Go back</a>
    </div>
  </div>
</div>

</main>

  <footer class="bg-dark text-white text-center py-4">
    <div class="container">
      <p>© 2025 Alejandro Rodríguez-Gijón</p>
    </div>
    <section class="text-center my-5">
  <h2>Other links</h2>
  <div class="d-flex justify-content-center gap-3 mt-3">
    <a href="https://github.com/alejandrorgijon" target="_blank" class="btn btn-dark">
      <i class="bi bi-github"></i> GitHub
    </a>
    <a href="https://x.com/alejandrorgijon" target="_blank" class="btn btn-info text-white">
      <i class="bi bi-twitter"></i> Twitter
    </a>
    <a href="https://bsky.app/profile/alejandrorgijon.bsky.social" target="_blank" class="btn btn-primary text-white">
       <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 530" width="20" height="20" fill="currentColor" class="me-1">
       <path d="M300 170.6c56.8-118 129.2-157.8 195.5-167.7 41.6-6.1 90.3 4.1 90.3 56.2 0 9.5-3.3 42.8-6.3 55.3-21.3 90.2-101.6 124.8-162.1 133.4 63.4 7.9 139.5 46.7 162.1 133.5 3 12.5 6.3 45.7 6.3 55.2 0 52.1-48.7 62.3-90.3 56.2-66.3-9.9-138.7-49.7-195.5-167.7-56.8 118-129.2 157.8-195.5 167.7-41.6 6.1-90.3-4.1-90.3-56.2 0-9.5 3.3-42.8 6.3-55.3 21.3-90.2 101.6-124.8 162.1-133.4-63.4-7.9-139.5-46.7-162.1-133.5C3.3 55.2 0 22 0 12.5 0-39.6 48.7-49.8 90.3-43.7c66.3 9.9 138.7 49.7 195.5 167.7z"/>
       </svg>
       Bluesky
    </a>        
    <a href="https://scholar.google.com/citations?user=5-3GnPQAAAAJ" target="_blank" class="btn btn-dark">
        <i class="bi bi-mortarboard-fill me-1"></i> Google Scholar
      </a>
    <a href="https://orcid.org/0000-0002-1649-6894" target="_blank" class="btn btn-success">
      <i class="fa-brands fa-orcid"></i> ORCID
      </a>
  </div>
</section>
  </footer>

  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>
  
</body>
</html>
